gradoop-store/gradoop-hbase/src/main/java/org/gradoop/storage/impl/hbase/factory/HBaseEPGMStoreFactory.java

```diff
@@ -150,9 +148,7 @@ private static void deleteEPGMStore(
     final String edgeTableName,
     final String graphTableName
   ) {
-    try {
-      Connection connection = ConnectionFactory.createConnection(config);
-
+    try (Connection connection = ConnectionFactory.createConnection(config)) {
```

Comment:
try-with-resources block closes the connection after successfully try block (or with failure in catch). This method has to keep the connection open for further store communication. 

I prepared a patch throwing an IOException instead of a try-catch block and passing it through. 
[#1244_add_exception.patch.zip](https://github.com/dbsleipzig/gradoop/files/3410478/1244_add_exception.patch.zip)